libbeat/tests/compose/wrapper.go

```diff
@@ -95,7 +98,47 @@ func (d *wrapperDriver) Up(ctx context.Context, opts UpOptions, service string)
 		args = append(args, service)
 	}
 
-	return d.cmd(ctx, "up", args...).Run()
```

Comment:
where was the retry before this change?

Reply:
I think I have been living too much in the world of #7957 :man_facepalming: 

I added a retry there because with the changes for multiple docker compose files and multiple versions the number of scenarios and thus the number of networks increases, what can end up taking all the address pools. On #7957 scenarios are started and destroyed on each test "suite", so this was a recoverable error.

I am going to close this PR by now, till the problem arises again.